### PR TITLE
Add specific Slack channel to request community review for initial pipeline releases

### PR DESCRIPTION
**Pull request by scwatts**
- indicate to readers a suitable channel to make a community review request
- this will help prevent requests being made in the wrong channels
  - e.g. #maintainers _hypothetically_ being mistakenly guessed as an appropriate channel

/cc @jfy133

@netlify /docs/contributing/tutorials/nf_core_contributing_overview

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]

**Comment by mashehu** [comment]
Could you please also add these changes to the following pr instead: https://github.com/nf-core/website/pull/2441
We will hopefully merge the PR soon and changes to the current docs might get lost on the way

**Comment by scwatts** [comment]
No problems. I don't have permissions to push to that branch but I'll rebase changes here on top + force push and then set it as the merge target.

**Comment by scwatts** [comment]
Looks like the Netlify Redirect check is required to merge even into a non-main branch. A few options I suppose:

- merge this change into `main` once `docs-restructure` is merged
- wait for the Netlify Redirect check failure to be fixed in `docs-restructure` before merging into it
- someone with write permissions cherry-pick the commit from my remote across to `docs-restructure`
- someone with write permissions manually make the change in `docs-restructure`

I'll leave it to the nf-core team to decide.

**Comment by scwatts** [comment]
The `docs-restructure` branch is now passing checks, rebasing on top and force pushing so that merge is possible.

**Comment by scwatts** [comment]
Automatically closed after deletion of `nf-core:docs-restructure`, opening a new PR since closing cannot be reversed.

**Pull request closed without merging**
